openapi/p4pa-pu-bff.openapi.yaml

```
@@ -3551,6 +3557,40 @@ paths:
           description: Not found
         '500':
           description: Internal server error
+  /organization/{organizationId}/dashboard-by-fc:
+    get:
+      tags:
+        - Dashboard
+      operationId: getInstallmentsDashboardByFiscalCode
+      parameters:
+        - name: organizationId
+          in: path
+          required: true
+          schema:
+            type: integer
+            format: int64
+        - name: fiscalCode
+          in: query
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/PagedDashboardDTO'
```

this is wrong... the result to return is 1... why a page?
```suggestion
                $ref: '#/components/schemas/DashboardByFc'
```